spec/numberformat.html

```diff
@@ -684,18 +684,21 @@ <h1>Properties of Intl.NumberFormat Instances</h1>
   <emu-clause id="sec-numberformat-abstracts">
     <h1>Abstract Operations for NumberFormat Objects</h1>
 
-    <emu-clause id="sec-currencydigits" type="abstract operation">
+    <emu-clause id="sec-currencydigits" type="implementation-defined abstract operation">
       <h1>
         CurrencyDigits (
           _currency_: a String,
         ): a non-negative integer
       </h1>
       <dl class="header">
+        <dt>description</dt>
+        <dd></dd>
       </dl>
+
       <emu-alg>
         1. Assert: IsWellFormedCurrencyCode(_currency_) is *true*.
         1. Assert: _currency_ is the ASCII-uppercase of _currency_.
```

Comment:
I don't think we need this assertion if we're not going directly against ISO 4217.
```suggestion
```

Reply:
Change